README.md

```diff
@@ -56,51 +51,88 @@ function Worker() {
+
+## Editing Worker Options
+As part of the [Worker API](https://docs.dcp.dev/specs/worker-api.html), the `worker` has a `workerOptions` property that describe the current active options configuring the worker, and mutating this object will modify worker options.
+
+__Note:__ To achieve desired React component updates regarding changes to certain options that may be featured in a UI, such as _paymentAddress_ - `worker.workerOptions` is a Proxy object with custom handling to handle component updates and writes to local storage.
+
+## Managing Origins
+
+The `worker` returned has the `originManager` property, which is an instance of the `OriginAccessManager` class responsible for managing the worker's allowed origins. `originManager` is `undefined` until the worker is properly initialized.
+
+Upon construction of the worker, the worker options `allowOrigins` property is read into the construction of the `OriginAccessManager`. Properties of `allowOrigins` translate to a _purpose_ on the OAM, with their values, being a list of origins, are added under that _purpose_ (and `null` _key_). The `any` property translates to a `null` _purpose_, which matches any _purpose_. For example, `isAllowed` will return `true` for origins stored under a `null` _purpose_, regardless the _purpose_ and _key_ combination queried.
+
+### OriginAccessManager Methods
+- `add(origin, purpose, key):` Adds (allows) the _origin_ under the _purpose_ and _key_. A `null` _purpose_ and/or _key_ will match any _purpose_ and/or _key_, respectively.
+- `remove(origin, purpose, key):` Removes (un-allows) the _origin_ under the _purpose_ and _key_. A `null` purpose and/or key will __not__ match any _purpose_ and/or _key_.
+- `remove_byKey(key):` Removes all origins for all purposes under the _key_. A `null` _key_ is not accepted, must be a string.
+- `getAllowList(purpose, key):` returns a list of origins under the _purpose_ and _key_. A `null` _purpose_ is not accepted, must be a string. Previously added origins under `null` _purpose_ and/or _key_ will match any _purpose_ and/or _key_, respectively.
+- `isAllowed(origin, purpose, key):` returns `true` if _origin_ is allowed under the _purpose_ and _key_. A `null` _purpose_ is not accepted, must be a string. Previously added origins under `null` _purpose_ and/or _key_ will match any _purpose_ and/or _key_, respectively.
+# Change log
+- __1.0.0__ - Feb 28, 2023
+  - Inital Release
+- __1.0.1__ - Mar 1, 2023
+  - Added src/ directory to published files to support source map
+- __1.0.2__ - Mar 23, 2023
+  - local storage will only save paymentAddress and maxWorkingSandbox props
+  - workerOptions source always coming from dcpConfig
+  - added delay between quick worker fetching states
+  - Quality of life + maintainability improvements
+- __1.1.0__ - May
```

Comment:
**suggestion:** This update is a major semver change as they break existing apps if people simply update their version and don't do anything else.

e.g., They'll need to migrate their usage of the hook from v1 to v2.

```suggestion
- __2.0.0__ - May
```

The version in `package.json` should also be changed.